pkg/api/validation/validation.go

```diff
@@ -1087,6 +1087,12 @@ func ValidatePersistentVolume(pv *api.PersistentVolume) field.ErrorList {
 	if numVolumes == 0 {
 		allErrs = append(allErrs, field.Required(specPath, "must specify a volume type"))
 	}
+
+	// do not allow hostPath mounts of '/' to have a 'recycle' reclaim policy
+	if pv.Spec.HostPath != nil && pv.Spec.HostPath.Path == "/" && pv.Spec.PersistentVolumeReclaimPolicy == api.PersistentVolumeReclaimRecycle {
```

Comment:
To properly guard here you'll need to do `path.Clean(pv.Spec.HostPath.Path)` because `/a/../` is also `/`.

Isn't this a slippery slope?  What about `/etc/`?  `/var`?  `/rootfs`?

Is this security or safety?